src/ProjectTemplates/GrainImplementation.NetStandard.CSharp/Company.Grains.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--/-:cnd -->
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Orleans.Core.Abstractions" Version="2.0.0-*" />
```

Comment:
Until we start being more careful for cross-versions compatibility in Orleans, I would just fix this to a concrete version, since behavior might change depending on when the user restores the packages, and potentially not be compatible across client and silo.